src/main/java/com/minecolonies/coremod/colony/jobs/AbstractJob.java

```diff
@@ -154,12 +150,7 @@ public static AbstractJob createFromNBT(final CitizenData citizen, @NotNull fina
      */
     public void readFromNBT(@NotNull final NBTTagCompound compound)
     {
-        final NBTTagList itemsNeededTag = compound.getTagList(TAG_ITEMS_NEEDED, Constants.NBT.TAG_COMPOUND);
-        for (int i = 0; i < itemsNeededTag.tagCount(); i++)
-        {
-            final NBTTagCompound itemCompound = itemsNeededTag.getCompoundTagAt(i);
-            itemsNeeded.add(new ItemStack(itemCompound));
-        }
+        //NOOP; Requests are stored on the building.
```

Comment:
:P It was one of the Corner Stones. Request are state based, so hey are stored in something else.